testing/i2pd/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Contributor: l-n-s <supervillain@riseup.net>
 # Maintainer: l-n-s <supervillain@riseup.net>
 pkgname=i2pd
-pkgver=2.24.0
-pkgrel=1
+pkgver=2.25.0
+pkgrel=0
 pkgdesc="I2P Router written in C++"
 url="https://github.com/PurpleI2P/i2pd"
 arch="all"
```

Comment:
license is not SPDX

Reply:
if you're adding openssl-dev then it brings openssl, so the openssl dependency is redundant

Reply:
has conf.d and init.d files but no $pkgname-openrc

Reply:
would be nice to have a comment on why check is disabled

Reply:
bulddir= is using default value so it can be removed

Reply:
no need for `cd "$builddir"`